src/command.ts

```diff
@@ -280,10 +284,18 @@ export class Command {
   runner(): (...a: any[]) => Promise<void> {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     return async (...args: any[]) => {
-      // always provide at least an empty object for options
-      if (args.length === 0) {
+      // Make sure the last argument is an object for options, add {} if none
+      if (typeof last(args) !== "object" || last(args) === null) {
```

Comment:
In what case will `last(args)` be (exactly) `null`? `last` returns `undefined` if there's nothing: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L7609

Should the second condition be `last(args) === undefined`? Or maybe just omitted, since `undefined` (or `null` for that matter) won't be an object?

Reply:
`typeof null === 'object'` and technically someone could do `firebase.apps.list(null)` so I wanted to defend against thinking that `null` was `{}`

Reply:
Oh you're right. Stupid javascript...